p2p/protocol/autonatv2/server.go

```diff
@@ -517,11 +517,11 @@ func (r *rateLimiter) Close() {
 
 // amplificationAttackPrevention is a dialDataRequestPolicy which requests data when the peer's observed
 // IP address is different from the dial back IP address
-func amplificationAttackPrevention(s network.Stream, dialAddr ma.Multiaddr) bool {
```

Comment:
dialAddr was previously unused? That seems like it should have been caught by a lint check.